source/library/Monadoc/Template/Package/Get.hs

```diff
@@ -28,39 +28,35 @@ data Input = Input
 
 render :: Context.Context -> Input -> Html.Html ()
 render context input = do
-  let packageName = Package.name . Model.value $ package input
+  let packageName = input.package.value.name
       route = Route.Package packageName
       title = F.sformat ("Package" F.%+ F.stext F.%+ ":: Monadoc") (Witch.from packageName)
-  Common.base context route (breadcrumbs input) title $ do
+  Common.base context route input.breadcrumbs title $ do
     Html.h2_ $ Html.toHtml packageName
     Html.h3_ "Uploads"
-    Html.ul_ . Monad.forM_ (rows input) $ \(upload Sql.:. version Sql.:. hackageUser Sql.:. _) -> Html.li_ $ do
-      let versionNumber = Version.number $ Model.value version
+    Html.ul_ . Monad.forM_ input.rows $ \(upload Sql.:. version Sql.:. hackageUser Sql.:. _) -> Html.li_ $ do
+      let versionNumber = version.value.number
```

Comment:
This whole file is a great example of how much nicer things look with this new syntax. There's a lot fewer parentheses and dollar signs.